scripts/services_java.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+function new_service() {
+    # $1 = service type
+    # $2 = port number
+    # $3 = hostname suffix
+    # $4 = additional command line arguments
+    typ=$1
+    port=$2
+    hostname_suffix=$3
+    arguments=$4
+
+    build_script="cd traffic-director-grpc-examples-master/java
+sudo apt-get install -y openjdk-11-jdk-headless
+./gradlew installDist"
+    server="./build/install/wallet/bin/${typ}-server"
+
+    startup_script="#! /bin/bash
+set -ex
+cd /root
+export HOME=/root
+sudo apt-get update -y
+curl -L https://storage.googleapis.com/traffic-director/td-grpc-bootstrap-0.9.0.tar.gz | tar -xz
+./td-grpc-bootstrap-0.9.0/td-grpc-bootstrap | tee /root/td-grpc-bootstrap.json
+curl -L https://github.com/GoogleCloudPlatform/traffic-director-grpc-examples/archive/master.tar.gz | tar -xz
+${build_script}
+sudo systemd-run -E GRPC_XDS_BOOTSTRAP=/root/td-grpc-bootstrap.json ${server} --port=${port} --hostname_suffix=${hostname_suffix} ${arguments}"
```

Comment:
Can this stuff be shared with the services_go script so it's not repeated?

Reply:
This is going to be bad for refactoring when it is inevitably going to happen.

If you need that requirement, maybe construct this file (and check it in) by concatenating others, and a comment saying "GENERATED CODE DO NOT EDIT"?

Or, just have one script with a parameter that says whether to run for java or go.